### PR TITLE
feat: add SEO for social links

### PR DESCRIPTION
**Pull request by wgwz**
## Description

Closes: https://github.com/regen-network/regen-registry/issues/1333

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] provided a link to the relevant issue or specification
- [ ] provided instructions on how to test
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed
- [ ] once the PR is closed, set up backport PRs for `redwood`, `hambach` and optionally `master` (see below)

#### Setting up backport PRs

After merging your PR to `dev`, set up backports by doing the following:

1. If your branch does not have merge commits, add the following comment to
   your PR:

- If PR is a new feature: `@Mergifyio backport redwood hambach`.
- If PR is a bug fix: `@Mergifyio backport redwood hambach master`.

2. If your branch does have merge commits:

   a. Pull latest `dev`, `hambach` and `redwood` branches

   b. Create new branches for backports and merge `dev` (replace `<PR#>` with your PR #)

   ```
   git checkout -b hambach-backport-<PR#> hambach
   git merge dev
   git push origin hambach-backport-<PR#>
   git checkout -b redwood-backport-<PR#> redwood
   git merge dev
   git push origin redwood-backport-<PR#>`
   ```

   c. Open new PRs in regen-web targeting `hambach` and `redwood`, respectively.

   d. If PR is a bug fix repeat the same process for `master` branch.

### How to test

1.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: web-registry/src/pages/Home/Home.tsx [hunk 1]
```diff
@@ -55,6 +57,17 @@ const Home: React.FC = () => {
 
   return (
     <Box sx={{ backgroundColor: 'primary.main' }}>
+      <SEO
+        title={seo?.title || ''}
+        description={seo?.description || ''}
+        imageUrl={seo?.image?.asset?.url || ''} // TODO: specify dimensions here. See: https://www.sanity.io/docs/image-urls
```
Comment (haveanicedavid) [inline web-registry/src/pages/Home/Home.tsx:63]: Just want to make sure you see this - maybe dont’ need to do now though

**Review by haveanicedavid — APPROVED** [review]
LGTM, though might want to double check the preview and see how big of an image it’s downloading (we probably on need something fairly small… quick google search says normally recommended resolution for an OG image is 1200 pixels x 627 pixels)

**Review by mhagel — APPROVED** [review]

File: web-registry/src/pages/Home/Home.tsx (hunk 1 above)
Comment (wgwz) [reply web-registry/src/pages/Home/Home.tsx:63]: it's not an ideal size, but it does seem to be working [with the browser extension check](https://socialsharepreview.com/browser-extensions) (for ref. for others if they see this). it's actually a bit smaller than the recommended size, so i think it's OK for now (1012x506):

https://cdn.sanity.io/images/jm12rn9t/staging/79a1cedb4c292dbfa50de8a681c67f642806170f-1012x506.png

**Comment by wgwz** [comment]
@Mergifyio backport redwood hambach master

**Pull request merged**